requirements/dev.txt

```diff
@@ -343,7 +343,7 @@ snowballstemmer==2.2.0
     #   -r requirements/test.txt
     #   pydocstyle
     #   sphinx
-sphinx==5.2.3
+sphinx==5.3.0
```

Comment:
No breaking change in the new version. For details see Changelog https://www.sphinx-doc.org/en/master/changes.html#release-5-3-0-released-oct-16-2022.